README.md

```diff
@@ -109,12 +113,19 @@ image[0].save("ddpm_generated_image.png")
 - [Unconditional Latent Diffusion](https://huggingface.co/CompVis/ldm-celebahq-256)
 - [Unconditional Diffusion with continous scheduler](https://huggingface.co/google/ncsnpp-ffhq-1024)
 
+**Other Notebooks**:
+* [image-to-image generation with Stable Diffusion](https://colab.research.google.com/drive/1yHBat74l_fvS9f4IDLvquvJSyY7G36G0?usp=sharing) ![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg),
```

Comment:
I think it's better to use a link to Suraj's repo instead of a (maybe ephemeral) google drive link:

```suggestion
* [image-to-image generation with Stable Diffusion](https://colab.research.google.com/github/patil-suraj/Notebooks/blob/master/image_2_image_using_diffusers.ipynb) ![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg),
```

Reply:
Ah today I learned you can open a github link in colab, this is 100x preferred (also an interpretable URL).